app/commands/fetch_datasource_expenses.py

```diff
@@ -82,105 +101,181 @@ async def fetch_datasources_for_organizations(
+                organizations = await organization_handler.query_db(
+                    where_clauses=[Organization.id == organization_id],
+                )
+            else:
+                logger.info("Querying organizations")
+                organizations = await organization_handler.query_db()
             logger.info("Found %d organizations to process", len(organizations))
 
-        async with OptscaleClient(settings) as optscale_client:
-            logger.info("Fetching datasources for the organizations to process from Optscale")
-            datasources_per_organization_id = await fetch_datasources_for_organizations(
-                organizations,
-                optscale_client,
-            )
-            logger.info("Completed fetching datasources for organizations")
+        for day, is_daily, frq in [
+            (today, False, "monthly"),
+            (yesterday, True, "daily"),
+        ]:
+            async with OptscaleClient(settings) as optscale_client:
+                logger.info(
+                    f"Fetching {frq} datasources expenses for the organizations from Optscale"
+                )
+                expenses = await fetch_datasource_expenses(
+                    organizations, optscale_client, day.year, day.month, day.day, is_daily=is_daily
+                )
+                logger.info(f"Completed fetching {frq} expenses")
 
-        async with session.begin():
-            logger.info(
-                "Storing datasource expenses for %s organiziations for %s",
-                len(organizations),
-                today.strftime("%d %B %Y"),  # e.g. "March 2025"
-            )
-            await store_datasource_expenses(
-                datasource_expense_handler,
-                datasources_per_organization_id,
-                year=today.year,
-                month=today.month,
-                day=today.day,
-            )
+            async with session.begin():
+                logger.info(
+                    "Storing %s datasource expenses for %s organiziations for %s",
+                    frq,
+                    len(organizations),
+                    today.strftime("%d %B %Y"),  # e.g. "20 March 2025"
+                )
+                await store_datasource_expenses(
+                    datasource_expense_handler,
+                    expenses,
+                    year=day.year,
+                    month=day.month,
+                    day=day.day,
+                    is_daily=is_daily,
+                )
 
-            logger.info("Completed storing datasource expenses")
+                logger.info(f"Completed storing {frq} datasource expenses")
 
 
-def command(ctx: typer.Context) -> None:
+def command(
+    ctx: typer.Context,
+    organization: Annotated[
+        str | None,
+        typer.Option(
+            "--organization",
+            "-o",
+            help="Organization ID (Default is all organizations)",
```

Comment:
We are using different ways to express defaults, sometimes we write

`Default: .....` Sometimes we enclose it within parentheses

We should fix it throughout the entire project (my personal preference is to omit parentheses).